interpreter.go

```diff
@@ -460,7 +463,7 @@ func unparseString(v string) string {
 	for _, c := range v {
 		switch c {
 		case '"':
-			buf.WriteString("\n")
+			buf.WriteString("\\\"")
```

Comment:
Huh, was the `\n` a prior bug or is something special going on? Your changed string makes sense.

Reply:
I think it was just a simple bug. There were no tests for that or anything. I don't see any reason to do that.